_data-prepper/log-analytics.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: Log analytics
+nav_order: 12
+---
+
+# Introduction
+
+Data Prepper is an extendable, configurable, and scalable solution for log ingestion into OpenSearch and Amazon OpenSearch Service. Data Prepper supports receiving logs from [FluentBit](https://fluentbit.io/) via the [Http Source](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/http-source/README.md), and processing those logs with a [Grok Processor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/grok-processor/README.md) before ingesting them into OpenSearch through the [OpenSearch sink](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/opensearch/README.md).
+
+Here are all of the components for log analytics with FluentBit, Data Prepper, and OpenSearch:
+
+![Log analytics component]({{site.url}}{{site.baseurl}}/images/data-prepper/log-analytics/log-analytics-components.png)
+
+In the application environment, run FluentBit. FluentBit can be containerized through Kubernetes, Docker, or Amazon ECS. You can also run FluentBit as an agent on EC2. Configure the [FluentBit http output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/http) to export log data to Data Prepper. Then, deploy Data Prepper as an intermediate component and configure it to send the enriched log data to your OpenSearch cluster. From there, use OpenSearch Dashboards to perform more intensive visualization and analysis. 
```

Comment:
Does it run as an agent specifically on EC2 or can it be run as an agent on any VM?

Reply:
That's a good question. @dlvenable do you have any input?

Reply:
Can be run on an agent with any VM.